util/delete-entities.py

```diff
@@ -0,0 +1,51 @@
+"""Delete all entities in a workspace.
+
+This is useful when working on Export to Saturn feature.
+
+To run:
+- virtualenv ~/virtualenv/firecloud
+- source ~/virtualenv/firecloud/bin/activate
+- pip install firecloud
+- python util/delete-entities.py WORKSPACE_NAMESPACE WORKSPACE_NAME
+"""
+
+import sys
+
+from firecloud import fiss
+
+
+def main():
+    if len(sys.argv) != 3:
+        raise ValueError(
+            'Usage: python util/delete-entities.py WORKSPACE_NAMESPACE WORKSPACE_NAME'
+        )
+
+    workspace_namespace = sys.argv[1]
+    workspace_name = sys.argv[2]
+
+    args = type('obj', (object, ), {
```

Comment:
I think the standard way to do this is by using a [namedtuple](https://docs.python.org/2/library/collections.html#collections.namedtuple) here:
```
Args = collections.namedtuple('Args', ['project', 'workspace'])
args = Args(workspace_namespace, workspace_name)
```

Though this is pretty neat, didn't realize it was possible this way.

Reply:
Done